### ISSUE TITLE
Return answer to single question 

### ISSUE THREAD

**Issue by ryancheley** [ryancheley/the-well-maintained-test#37]
**Is your feature request related to a problem? Please describe.**
When running `the-well-maintained-test questions -q 1` it would be nice if in addition to the question and context of the question, the response to the question was return as well. 

**Describe the solution you'd like**
When running `the-well-maintained-test questions -q 1 --name the-well-maintained-test` The following would be return

```
1. Is it described as “production ready”?
                                                                                
    We want to see evidence that the maintainers consider the software as ready 
    for use in production.                                                      
                                                                                
    The documentation shouldn’t have any banners or wording implying a future   
    stable release.                                                             
                                                                                
    The version number should not be a pre-release, alpha, beta, release        
    candidate, etc. Note that some maintainers stick with a “zero version       
    number” like 0.4.0, even when they consider the package production ready.   
                                                                                
                                                                                
See https://adamj.eu/tech/2021/11/04/the-well-maintained-test/#is-it-described-a
s-production-ready for the original source.

The project is set to Development Status Beta
```

**Describe alternatives you've considered**
There are none

**Additional context**
None

**Comment by ryancheley** [comment]
Pushed up initial implementation with 58f6d5f. 

Still to do:

- [ ] Fix / update how question 3 is asked. It currently takes two input parameters that are responses. All of the other methods take in `url` as `str` and this discrepancy is causing an issue with the update to the `questions.toml`which includes the url to use 🤦‍♂️

**Comment by ryancheley** [comment]
Question 3 was updated, it's just the tests that need to be fixed. May be time to take a step away 😄

**Comment by ryancheley** [comment]
OK ... last comment. I think I need to update the method to look at the `Changelog` in `project_urls` and that will take care of the problem completely ... maybe 🤷‍♂️

**Comment by ryancheley** [comment]
closed with 96d2173
